static/src/inline-svgs/illustration/my-digest.svg

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This is what @NathanielBennett had mentioned. Seeing as it is just a 404 page for now, it can stay as is.